src/theme.js

```diff
@@ -29,6 +29,29 @@ const theme = createTheme({
       main: '#64748B',
       contrastText: '#fff'
     }
+  },
+  // color properties TBD for status snackbar severieties
+  components: {
```

Comment:
Could you go over your thoughts for this theme a bit more? I'm a little confused how it's conceptually different from the "status" section higher up.

Reply:
The way I understand it,  the top section of `theme` is mostly `palette`(line 5) - which is general colors.
this new section is `components` where we can override the base theme of any components we want, so we don't have to say `color='primary'` every time. It will just put that primary color on it automatically.

So here, it looks like Russ is overriding the theme colors used for specifically the `MuiAlert` (which we'll need to change to just `Alert` if he moves forward with my recommendations/comments).  So we can make the specific type of `Alerts` any color we wish by setting it once here and they will automatically be that. 

So if UX team says success alerts should be purple, and error alerts should be pink - we set that here and anytime we put `<Alert severity='success'>` in the codebase it will be purple. No need to do `<Alert severity='success' color='purple'>`

Reply:
This will make it a bit confusing to reuse the colors across different components. We may use these warning / error / info colors in e.g. form validations.

MUI palettes suggest having `warning`, `error`, and `info` fields as part of the palette itself. I think we can structure the theme that way, and all our cases should be covered.

Take a look at this link: https://mui.com/material-ui/customization/palette/#default-colors

Reply:
Sorry for the late reply on this. Would the answer be to keep the Mui alert overides for the convenience of just being able to set an alert type and have it be the correct color, but then also storing the status colors in the status object for use in other places? The colors would be repeated, but it would be nice not to have to set the colors on every alert as well.